packages/angular_devkit/core/src/json/parser_spec.ts

```diff
@@ -25,15 +25,13 @@ describe('parseJson and parseJsonAst', () => {
         expect(ast.start).toEqual({ offset: start[0], line: start[1], character: start[2] });
         expect(ast.end).toEqual({ offset: end[0], line: end[1], character: end[2] });
         expect(ast.value).toEqual(JSON.parse(n));
-        expect(parseJson(n)).toEqual(JSON.parse(n));
         expect(ast.text).toBe(text === undefined ? n : text);
       });
     }
 
     for (const n of errors) {
       it(`errors for ${JSON.stringify(n)}`, () => {
         expect(() => parseJsonAst(n)).toThrow();
```

Comment:
Removing `parseJsonAst` is much more involved since the workspace is highly dependent on this form of AST.